webviz_subsurface/plugins/_simulation_time_series/_plugin.py

```diff
@@ -267,7 +269,7 @@ def __init__(
             self._initial_vector_selector_data, self._predefined_expressions
         )
 
-        plot_options = options if options else {}
+        plot_options = {} if options is None else options
```

Comment:
see that I was a bit fast and rewrote it differently, will change back. No need to change this line 😅